### DISCUSSION TITLE
Inconsistent behaviour of substition variable name in output.assetModuleFilename

### DISCUSSION THREAD

**Discussion by bkilinc** [webpack/webpack#14766] [General]
# Bug report
I have multiple entry points like 
```
    entry: {
      mod1: './src/modules/mod1.js',
      mod2: './src/modules/mod2.js',
      mod3: './src/modules/mod3.js'
    },
```
I want to create subfolders for each entry point, emit all related files (assets etc.) into the folder. this works  perfectly for main output file. like this.
```
    output: {
      path: __dirname + '/assets',
      filename: '[name]/bundle.js'
    },
```
at the end I have mod1, mod2, mod3 subfolders with bundle.js in them. I also want to emit assets into subfolders. Using same substitution names.
```
    output: {
      path: __dirname + '/dist',
      filename: '[name]/bundle.js',
      assetModuleFilename: '[name]/assets/[hash][ext][query]'
    },
```
this does not work. the 'name' variable does not represent same thing.

**What is the current behavior?**

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**

**Other relevant information:**
webpack version:  5.60.0
Node.js version: 14
Operating System: Linux Mint 19
Additional tools:

**Comment by alexander-akait** [comment]
Sorry, assets are not related to entrypoints
